doc/installation.rst

```diff
@@ -23,7 +23,7 @@ Quick & dirty
 
 Start the web UI (will spawn a HTTP server on ``http://localhost:5000``)::
 
-  $ ./yeti.py
+  $ ./yeti.py webserver
```

Comment:
The "syncdb" or "apply_migrations" or whatever operation should also be mentioned in the documentation. In a new update section ?